ls1intum/Artemis
# `Development`: Add exam signature upload test

### Checklist
#### General


#### Server

### Motivation and Context
**Previously**, the Exam Checker App couldn't load the signature after it was submitted. The Test checks that the behavior stays fixed.

![Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2023-09-20 at 01 29 16](https://github.com/ls1intum/Artemis/assets/23535000/686336a3-6997-4a78-86a6-98fb5b160034)

### Description


### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student
- 1 Exam

1. Log in to the Exam Checker App
2. Let _student1_ sign their information
3. Verify that the app can load/display the signature

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests
#### Exam Mode Test
_Planned on Monday_

### Test Coverage
